independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanDeployment.java

```diff
@@ -64,6 +64,9 @@
 
     private final Map<DotName, StereotypeInfo> stereotypes;
 
+    // excluding additional BeanDefiningAnnotations
+    private final Map<DotName, StereotypeInfo> realStereotypes;
```

Comment:
I don't think that we need to store this map in a field. It could be just a local variable inside the `findBeans()` method. WDYT?

Reply:
Yeah I wasn't quite sure, then I thought it might be useful in some other methods potentially in the future. But maybe not. I'll turn it into a local variable for now